contracts/strategies/dnsVectorStrategy/DnsVectorStrategyAumLib.sol

```diff
@@ -128,13 +126,9 @@ library DnsVectorStrategyAumLib {
             uint112 reserve0, // wAvax (generally it can be aaveSupplyToken or aaveBorrowToken) // usdc (generally it can be aaveSupplyToken or aaveBorrowToken)
```

Comment:
getPoolDebt is a real bad naming from me, Pool can refer to 
LendingPool or LiquidityPool. Feel free to change it, otherwise I will change it in a later release. (I don't think the offchain component depends on this)

Reply:
Ok, I'll leave it.